CRM/Contact/BAO/SavedSearch.php

```diff
@@ -395,12 +395,14 @@ public static function getSmartGroupsUsingGroup(int $groupID) {
     $smartGroups = [];
     foreach ($groups as $group) {
       // Filter out arrays in Form Values which are group searchs.
-      $groupSearches = array_filter(
-        $group['saved_search_id.form_values'],
-        function($v) {
-          return ($v[0] == 'group');
-        }
-      );
+      if (is_array($group['saved_search_id.form_values'])) {
```

Comment:
Looks like the `$groupSearches` variable is now possibly undefined & we need to add this:
```suggestion
      $groupSearches = [];
      if (is_array($group['saved_search_id.form_values'])) {
```

Also, this PR should probably be against the 5.75 branch.

Reply:
Good suggestion. I've made the change in the source code and recommitted.